federatedscope/core/fed_runner.py

```diff
@@ -208,6 +210,12 @@ def _setup_client(self, client_id=-1, client_model=None):
 
         if self.client_class:
             client_specific_config = self.cfg.clone()
+            # Modify the setting according to self.cfg_client in standalone mode
+            if self.cfg_client is not None:
+                client_specific_config.defrost()
+                client_specific_config.merge_from_other_cfg(self.cfg_client.get('client_{}'.format(client_id)))
+                client_specific_config.freeze()
```

Comment:
The `freeze()` will override the cfg.yaml file. We may need to add a parameter for this function to skip the save process

Reply:
It is modifed accordingly.